src/examples/cameras.cpp

```diff
@@ -54,7 +54,7 @@ int main()
     graphics->record_video("video-main.mp4", simu.graphics_freq());
 
     // Add camera
-    auto camera = std::make_shared<robot_dart::gui::magnum::CameraOSR>(&simu, graphics->magnum_app(), 256, 256);
+    auto camera = std::make_shared<robot_dart::gui::magnum::sensor::Camera>(&simu, graphics->magnum_app(), 256, 256);
```

Comment:
It sounds weird to have a sensor in the gui namespace...

Reply:
This is true.. I'll try to make the camera sensors headers to be independent of `Magnum` and only the `cpp` files to require `Magnum`.